sdk/keyvault/keyvault-keys/src/localCryptographyClient.ts

```diff
@@ -105,21 +107,13 @@ export class LocalCryptographyClient {
    */
   public key: JsonWebKey;
 
-  /**
-   * Constructs a new instance of the Local Cryptography client for the given key.
-   *
-   * Example usage:
-   * ```ts
-   * import { LocalCryptographyClient } from "@azure/keyvault-keys";
-   *
-   * const jsonWebKey: JsonWebKey = {
-   *   // ...
-   * };
-   * const client = new LocalCryptographyClient(jsonWebKey);
-   * ```
-   * @param key - The JsonWebKey to use during cryptography operations.
-   */
   constructor(key: JsonWebKey) {
     this.key = key;
   }
+
+  public vaultUrl: undefined;
```

Comment:
Maybe it doesn't matter because this is now internal - sort of - but shouldn't this still be a function that returns `undefined` for call-compatibility?

Reply:
From the consumer side both will be called with `client.vaultUrl` - they're both getters and get translated to readonly properties (since there's no setter), one just looks differently because it's a computed getter. But.... you have me thinking about this now. If there's confusion maybe it would be easier to grok if I rewrote this as a computed getter. `get vaultUrl() { return undefined }` or something and I can add some internal docs about why its undefined.